docs/ja/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,223 @@
+# カスタムレスポンス - HTML、ストリーム、ファイル、その他のレスポンス
+
+デフォルトでは、**FastAPI** は `JSONResponse` を使ってレスポンスを返します。
+
+[レスポンスを直接返す](response-directly.md){.internal-link target=_blank}で見たように、 `Response` を直接返すことでこの挙動をオーバーライドできます。
+
+しかし、`Response` を直接返すと、データは自動的に変換されず、ドキュメントも自動生成されません (例えば、生成されるOpenAPIの一部としてHTTPヘッダー `Content-Type` に特定の「メディアタイプ」を含めるなど) 。
+
+しかし、*path operationデコレータ* に、使いたい `Response` を宣言することもできます。
+
+*path operation関数* から返されるコンテンツは、その `Response` に含まれます。
+
+そしてもし、`Response` が、`JSONResponse` や `UJSONResponse` の場合のようにJSONメディアタイプ (`application/json`) ならば、データは *path operationデコレータ* に宣言したPydantic `response_model` により自動的に変換 (もしくはフィルタ) されます。
+
+!!! note "備考"
+    メディアタイプを指定せずにレスポンスクラスを利用すると、FastAPIは何もコンテンツがないことを期待するので、生成されるOpenAPIドキュメントにレスポンスフォーマットが記載されません。
```

Comment:
```suggestion
    メディアタイプを指定せずにレスポンスクラスを利用すると、FastAPIは何もコンテンツがないことを期待します。そのため、生成されるOpenAPIドキュメントにレスポンスフォーマットが記載されません。
```

This is a long sentence, so separating the two sentences will make it easier to read.